apps/web/src/pages/ManagerDashboardPage/messages.ts

```diff
@@ -0,0 +1,17 @@
+import { defineMessages } from "react-intl";
+
+const messages = defineMessages({
+  pageTitle: {
+    defaultMessage: "Manager dashboard",
+    id: "o1jXs5",
+    description: "title for the 'Manager dashboard' page",
+  },
+  subTitle: {
+    defaultMessage:
+      "Manage your talent searches and curate your manager profile.",
+    id: "lYp2e8",
+    description: "subtitle for the 'Manager dashboard' page",
+  },
+});
```

Comment:
```suggestion
const messages = defineMessages({
  pageTitle: {
    defaultMessage: "Manager dashboard",
    id: "o1jXs5",
    description: "Title for the 'Manager dashboard' page",
  },
  subTitle: {
    defaultMessage:
      "Manage your talent searches and curate your manager profile.",
    id: "lYp2e8",
    description: "Subtitle for the 'Manager dashboard' page",
  },
});
```

Reply:
Updated in [update message descriptions](https://github.com/GCTC-NTGC/gc-digital-talent/pull/11342/commits/ec5608223c84ab431ae770d10df0c58026c7c9f9)